__tests__/building-highlights.test.js

```diff
@@ -1,12 +1,38 @@
 import React from "react";
 import BuildingHighlights from "../src/components/building-highlights/building-highlights.component";
 import renderer from "react-test-renderer";
+import { shallow } from "enzyme";
+import { BuildingId } from "../src/types/main";
+import { Buildings } from "../src/constants/buildings.data";
+import { CONCORDIA_RED, BUILDING_UNTAPPED } from "../src/constants/style";
 
-// Kind: Snapshot + Unit Tests
-// [US-2] Checks that all the highlights are rendering at their proper coordinates (latitude and longitude) also with proper styling
-describe("Building Highlight Snapshop", () => {
-  test("renders correctly", () => {
+describe("BuildingHighlights component", () => {
+  it("BuildingHighlights: snapshot renders correctly", () => {
```

Comment:
Like @osahtout mentionned:

Formulate the string such that it flows4 with the word "it" and how you expect the component to behave:
Example:
"it" "should match the snapshot"
it "should change the polygon color and call onBuildingTap after press"